Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -351,8 +351,11 @@ function GetMatchingComments
             ($_.ItemPath -eq $message.RelativePath) -and
             (ThreadMatchesCommentSource $_ $script:messageSource)}
 
-     Write-Verbose "Found $($matchingThreads.Count) matching thread(s) for the message at $($message.RelativePath) line $($message.Line)"
-        
+     if ($matchingThreads.Count -gt 0)
```

Comment:
Well, if you post hundreds of new messages, you'll get hundreds of these messages. No matching thread means that a comment is new